src/main.rs

```diff
@@ -31,7 +31,7 @@ mod tests;
 
 #[tokio::main]
 async fn main() {
-    println!("starknetid_server: starting v{}", env!("CARGO_PKG_VERSION"));
+    // println!("starknetid_server: starting v{}", env!("CARGO_PKG_VERSION"));
```

Comment:
Don't comment this line. Instead, move it after the logger definition, and use it